Adding pvc and block device notes. (#565)
rh-ecosystem-edge/ztp-pipeline-relocatable: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc

    . 1 for the HTTPD server, which will host the RHCOS images.
    . We need to meet the OpenShift Storage requirements for the Hub like (SSD/NVME).
    . LSO should be enough but we recommend to use a more reliable storage backend like ODF or NFS in order to avoid issues with the PVs and node scheduling pods.

> - We need an existing PVC named ztp-pvc to be successfully bound to a PV. You can use the following yaml to create the PVC.

Reviewer: ```suggestion
- Create a PVC called `ztp-pvc` that will be used by the hub pipeline itself. You can use the following yaml to create the PVC.
```

```
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  annotations:

>   finalizers:
>   - kubernetes.io/pvc-protection

Reviewer: ```suggestion
```

Author: what is different?

Reviewer: Remove that section

  name: ztp-pvc
  namespace: edgecluster-deployer
spec:
  accessModes:
  - ReadWriteOnce
  resources:
    requests:
      storage: 10Gi

>   storageClassName: standard

Reviewer: ```suggestion
```

Author: not clear what differs?

Reviewer: Remove the storage class

    			--use-param-defaults hub-deploy-lvmo \
    			--showlog
```

>     NOTE:: You'll need to make sure that the HUB's master nodes have one available block device to form the required volume group. The total size of these block devices should be no less than 900GB.

Reviewer: I would rephrase this as: 

Please, make sure to have an available, default, storageclass configured in the hub cluster for the PVC to be properly bound.

Author: This note is for the automated PV creation - we need the block devices, right?

Reviewer: ah, good point. The diff confused me and I thought it was a generic note. It's correct, then.
